client/reader/following-manage/search-followed.jsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * External Dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import SearchCard from 'components/search-card';
+
+class FollowingManageSearchFollowed extends Component {
+	static propTypes = {
+		value: PropTypes.string,
+		onSearch: PropTypes.func,
+	}
+
+	static defaultProps = {
+		onSearch: noop,
+	}
+
+	render() {
+		return (
+			<SearchCard
+				compact={ true }
+				isOpen={ true }
+				pinned={ false }
+				autoFocus={ false }
+				className="following-manage__search-followed"
+				additionalClasses="following-manage__search-followed-input"
+				placeholder={ this.props.translate( 'Search Followed Sites…' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Search your followed sites' )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/reader/following-manage-search-following-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).